tests/beamlines/unit_tests/test_beamline_utils.py

```diff
@@ -69,7 +69,7 @@ def test_instantiate_v2_function_fake_makes_fake():
 
 def test_clear_devices(RE):
     devices = make_all_devices(i03, fake_with_ophyd_sim=True)
-    assert len(beamline_utils.ACTIVE_DEVICES) == len(devices.keys())
+    assert len(beamline_utils.ACTIVE_DEVICES) == len(devices[0].keys())
```

Comment:
Nit: You could instead have

```python
devices, exceptions = make_all_devices(i03, fake_with_ophyd_sim=True)
assert len(beamline_utils.ACTIVE_DEVICES) == len(devices.keys()) and len(exceptions) == 0
```

...which is more readable, ditto for the other tests